webrecorder/browsertrix
# Update tab access by user role

Hides "Crawl Configs" and "Browser Profile" for viewer roles, + some minor type file reorganizing. Resolves https://github.com/webrecorder/browsertrix-cloud/issues/525.

### Manual testing
1. Log in as a user with a "viewer" role. Verify only the "Crawls" top-level navigation tab is shown.
2. Log out and log in as user with "crawler" role. Verify "Crawls", "Crawl Configs", and "Browser Profiles" is shown.
3. Log out and log in as an org owner. Verify all crawler tabs + "Org Settings" is shown.
